src/python/requirements-app.txt

```diff
@@ -1,2 +1,16 @@
 ### your custom requirements here
-#protobuf==3.13.0
\ No newline at end of file
+black
+flake8
+graphistry[ai]
+ipython
+isort
+jupyter
+mypy
+natsort
+networkx
```

Comment:
surprised we need networkx in the new demos, + i think it's already in the base image